docs/pages/developer-guide.md

```diff
@@ -1775,7 +1775,9 @@ pods:
         resource-set: hello-resources/
 ```
 
-In this case, the resources are declared separately from the server task in a resource set named `hello-resources`. They are referenced by a `resource-set` element in the task definition. A task continues to be defined as the combination of a process to run and resources to consume. This alternate formulation provides you with increased  flexibility:  you can now define multiple processes that can consume the same resources.
+In this case, the resources are declared separately from the server task in a resource set named `hello-resources`. They are referenced by a `resource-set` element in the task definition. A task continues to be defined as the combination of a process to run and resources to consume. This alternate formulation provides you with increased flexibility: you can now define multiple processes that can consume the same resources.
+
+**Note:** Resource sets are reserved at scheduler startup. [Custom auxiliary plans](#custom-auxiliary-plans) that run tasks that make use resource sets won't require additional resources than what the scheduler already reserved.
```

Comment:
In terms of style, there seem to be a lot of "that"s in the second sentence.

Reply:
What do you think about
> Custom auxiliary plans that run tasks referencing resource sets won't require [...]

Reply:
Sounds good 🎵